en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+         <body>
+             <img src="../../assets/img/Learn/subscription-commercial-tier.png" alt="Subscription to a commercial tier" title="Commercial tier subscription" width="400" />
+         </body>
+     </html>
+
+     After you save the policy, a plan gets created in the Stripe account of the tenant admin.  
+
+     ![Stripe account after creating a commercial tier](../../assets/img/Learn/Stripe-account-after-creating-a-commercial-tier.png)  
+     
+     The newly created Stripe product ID and plan ID will be stored in `AM_POLICY_PLAN_MAPPING` table along with the `UUID` of the tier. When you update the details of this tier, the plan in Stripe will get updated with the corresponding details. Likewise, when you delete this tier, the plan in Stripe will get deleted and the corresponding record in the DB will also get deleted.
+
+#### Step 2 - Enable monetization
+
+1.  <a name> additionalProp </a> Configure the additional properties that are specific to the billing engine with regard to monetization in WSO2 API Manager.  
+     
+     After saving these configurations, the properties get populated in the Monetization enable/disable page on the Publisher in WSO2 API Manager.  
+
+    1.  Add the following configuration under Monetization in the `<API-M_HOME>/repository/resources/conf/templates/repository/conf/api-manager.xml.j2` file.
+
+        ``` java
+        <AdditionalProperties>
+        {% for origin in apim.monetization.additional_properties %}
+                <Property>{{origin}}</Property>
+        {% endfor %}
+        </AdditionalProperties>
+
+        {% if apim.monetization.additional_attributes is defined %}
+                    <AdditionalAttributes>
+                        {% for attribute in apim.monetization.additional_attributes %}
+                        <Attribute required="{{attribute.required}}">
+                            <Name>{{attribute.name}}</Name>
+                            <DisplayName>{{attribute.display_name}}</DisplayName>
+                            <Description>{{attribute.description}}</Description>
+                            <Default>{{attribute.default}}</Default>
+                        </Attribute>
+                        {% endfor %}
+                    </AdditionalAttributes>
+        {% endif %}
+        ```
+
+    2.  Add the following configuration in the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+        ``` java
+        [apim.monetization]
+        additional_attributes=['ConnectedAccountKey']
+        ```
+
+        When using Stripe as the billing engine, you need to enter the connect ID, which is the ID that indicates the link between the tenant admin and API provider Stripe accounts.
+
+2.  Enable API monetization.
+
+    1.  Sign in to the WSO2 API Manager Publisher.  
+        `https://<hostname>:9443/publisher-new`
+    2.  Click on the API that you wish to monetize.  
+        ![Enable monetization](../../assets/img/Learn/enable-monetization.png)
```

Comment:
Image path needs to be corrected

Reply:
Fixed.
I added the path based on the example on page 6 in the "Guidelines for Github Documentation" document. Looks like I misinterpreted the example.